setup.py

```diff
@@ -0,0 +1,15 @@
+import setuptools
+
+import lodge
+
+
+VERSION = lodge.__version__
```

Comment:
Maybe he is making the version provided in a unique place where it's possible to consult at log runtime, that way it's possible to log the lodge version itself...?

Reply:
@ryukinix lodge version is not used during logging. The application version - that uses the lodge - should be used on the logging.

Reply:
However, that is a common pattern on some open source software: embedded on software the version and maintain the version in the source code.

There is some downsides, like: we should take care on release the tag, always updating before the source code version. That makes some Continuoues Delivery operations more complicated: like releasing automatic software to PyPI via github tags.

We should listen what @mfrata had in mind using this pattern, I used a lot in the past on my open source software which I maintain.

Reply:
This is to centrilize the version of the package in the __version__ variable. A lot of  libs use this way. One example is the [pandas lib](https://github.com/pandas-dev/pandas/blob/2019e7b71f392a2790d95fa26b11fe917946e1d1/pandas/_version.py#L511), that uses versioneer to manage the versioning of its project. 

Anyway, since this is a simple lib we can move it to the setup.py.